src/test/kotlin/com/terraformation/backend/documentproducer/api/DocumentTemplatesControllerTest.kt

```diff
@@ -7,7 +7,7 @@ import org.junit.jupiter.api.Test
 import org.springframework.test.web.servlet.get
 
 class DocumentTemplatesControllerTest : ControllerIntegrationTest() {
-  private val path = "/api/v1/document-templates"
+  private val path = "/api/v1/document-producer/document-templates"
```

Comment:
Is this a bit too verbose? Is `...document-producer/templates` OK here?

Reply:
I think `document-producer/templates` is fine. No ambiguity with templates in other parts of the API since it's under `document-producer`.